contentcuration/contentcuration/frontend/channelList/views/Channel/StudioMyChannels.vue

```diff
@@ -0,0 +1,582 @@
+            () => {
+              this.deleteDialog = false;
+              this.$store.dispatch('showSnackbarSimple', this.$tr('channelRemovedSnackbar'));
+            },
+          );
+        } else {
+          this.deleteChannel(this.selectedChannel.id).then(() => {
+            this.deleteDialog = false;
+            this.$store.dispatch('showSnackbarSimple', this.$tr('channelDeletedSnackbar'));
+          });
+        }
+      },
+      thumbnailSrc(channel) {
+        return channel.thumbnail_encoding && channel.thumbnail_encoding.base64
+          ? channel.thumbnail_encoding.base64
+          : channel.thumbnail_url;
+      },
+      getPublishStatus(channel) {
+        return channel.last_published
+          ? this.$tr('lastPublished', {
+            last_published: this.$formatRelative(channel.last_published, {
+              now: new Date(),
+            }),
+          })
+          : this.$tr('unpublishedText');
+      },
+    },
+    $trs: {
+      channel: 'New channel',
+      noChannelsFound: 'No channels found',
+      resourceCount: '{count, plural,\n =1 {# resource}\n other {# resources}}',
+      unpublishedText: 'Unpublished',
+      lastPublished: 'Published {last_published}',
+      lastUpdated: 'Updated {updated}',
+      details: 'Details',
+      viewContent: 'View channel on Kolibri',
+      goToWebsite: 'Go to source website',
+      editChannel: 'Edit channel details',
+      copyToken: 'Copy channel token',
+      deleteChannel: 'Delete channel',
+      deleteTitle: 'Delete this channel',
+      removeChannel: 'Remove from channel list',
+      removeBtn: 'Remove',
+      removeTitle: 'Remove from channel list',
+      deletePrompt: 'This channel will be permanently deleted. This cannot be undone.',
+      removePrompt:
+        'You have view-only access to this channel. Confirm that you want to remove it from your list of channels.',
+      channelDeletedSnackbar: 'Channel deleted',
+      channelRemovedSnackbar: 'Channel removed',
+      channelLanguageNotSetIndicator: 'No language set',
+      cancel: 'Cancel',
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .window-small {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+    }
+  }
+
+  .window-medium {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 83.33%;
+    }
+  }
+
+  .window-large {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 50%;
+    }
+  }
+
+  .windowBreakpoint-3 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 83.33%;
+    }
+  }
+
+  .windowBreakpoint-4 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 66.66%;
+    }
+  }
+
+  .windowBreakpoint-5 {
+    .channels-body,
+    .new-channel {
+      width: 100%;
+      max-width: 50%;
+    }
+  }
+
+  .my-channels {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: calc(100vh - 64px - 48px);
```

Comment:
One possible concern here is that `vh` is problematic on mobile devices: https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser - better to use `100%` instead unless there's a compelling reason not to.